CHANGELOG.md

```diff
@@ -43,6 +43,8 @@
 
 #### Changed
 - Upgraded to IDE Rider v3.0.28 (#166)
+- Upgraded to Unity 2022.3.27f1 (#176)
+  - com.unity.transport upgraded to v1.4.1
 
```

Comment:
what about the burst package, should it not be mentioned?

Reply:
I'm not attached to leaving it out or not.
It's just not included inside the sample's manifest and it's not a package relating to Netcode, Tools, or Services, so to reduce noise on upgrades we should highlight core upgrades.

Reply:
OK!